media-video/simplescreenrecorder/simplescreenrecorder-9999.ebuild

```diff
@@ -94,6 +94,7 @@ multilib_src_configure() {
 		$(multilib_native_use_with pulseaudio)
 		$(multilib_native_use_with jack)
 		$(multilib_native_use_with qt5)
+		$(use_enable asm x86_asm)
```

Comment:
Currently it attempts to use ./configure .... --disable-x86_asm. From ./configure --help:
````
  --disable-x86-asm       don't use x86/x64 assembly instructions
````